src/components/core/base/test/GeoValidatorHOC.spec.tsx

```diff
@@ -152,4 +140,41 @@ describe("GeoValidatorHOC", () => {
             },
         ]);
     });
+
+    it("should show GEO_MAPBOX_TOKEN_MISSING error", () => {
```

Comment:
please move these new tests next to "should show GEO_LOCATION_MISSING error" test

Reply:
fixed